### PR TITLE
IGVF-802-rename-references

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/mixins.json [hunk 1]
```diff
@@ -292,16 +292,16 @@
...
+        "publication_identifiers": {
+            "title": "Publication Identifiers",
+            "description": "The publication identifiers that provide more information about the object.",
             "type": "array",
             "minItems": 1,
             "uniqueItems": true,
             "items": {
-                "title": "Reference",
-                "description": "A reference that provides more information about the object.",
+                "title": "Publication Identifiers",
```
Comment (gabdank) [inline src/igvfd/schemas/mixins.json:303]: Should be singular rather than plural

File: src/igvfd/tests/test_upgrade_auxiliary_set.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def test_auxiliary_set_upgrade_1_2(upgrader, auxiliary_set_v1):
+    value = upgrader.upgrade(
+        'auxiliary_set', auxiliary_set_v1,
+        current_version='1', target_version='2')
+    assert value['schema_version'] == '2'
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_auxiliary_set.py:8]: please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost

File: src/igvfd/tests/test_upgrade_construct_library.py [hunk 3]
```diff
@@ -17,3 +17,12 @@ def test_construct_library_upgrade_2_3(upgrader, construct_library_v2):
     assert value['schema_version'] == '3'
     assert 'origins' in value
     assert 'guide_library_details' in value
+
+
+def test_construct_library_upgrade_3_4(upgrader, construct_library_v3):
+    value = upgrader.upgrade(
+        'construct_library', construct_library_v3,
+        current_version='3', target_version='4')
+    assert value['schema_version'] == '4'
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_construct_library.py:26]: please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost

File: src/igvfd/tests/test_upgrade_curated_set.py [hunk 4]
```diff
@@ -16,3 +16,12 @@ def test_curated_set_upgrade_2_3(upgrader, curated_set_v2):
...
     assert value['accession'] == 'IGVFDS0000ZZZA'
     assert value['schema_version'] == '3'
+
+
+def test_curated_set_upgrade_3_4(upgrader, curated_set_v3):
+    value = upgrader.upgrade(
+        'curated_set', curated_set_v3,
+        current_version='3', target_version='4')
+    assert value['schema_version'] == '4'
+    assert 'publication_identifiers' in value
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_curated_set.py:26]: please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost

File: src/igvfd/tests/test_upgrade_model.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def test_model_upgrade_1_2(upgrader, model_v1):
+    value = upgrader.upgrade(
+        'prediction', model_v1,
+        current_version='1', target_version='2')
+    assert value['schema_version'] == '2'
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_model.py:8]: please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost

File: src/igvfd/upgrade/sample.py [hunk 6]
```diff
@@ -248,3 +248,20 @@ def in_vitro_system_11_12(value, system):
     if 'time_post_factors_introduction_units' in value:
         value['time_post_change_units'] = value['time_post_factors_introduction_units']
         del value['time_post_factors_introduction_units']
+
+
+@upgrade_step('whole_organism', '14', '15')
```
Comment (gabdank) [inline src/igvfd/upgrade/sample.py:253]: was that missing? why is it added in this PR?

**Review by gabdank — CHANGES_REQUESTED** [review]
see comments on PR

File: src/igvfd/upgrade/sample.py (hunk 6 above)
Comment (ian-whaling) [reply src/igvfd/upgrade/sample.py:253]: I just noticed whole_organism.py was created with this single upgrade: https://github.com/IGVF-DACC/igvfd/pull/470/files#diff-acbed1fca8facc3ec3a83224b0d74c38c268cd5635a656c8437022c1bdcdd389 and decided to just move it to the shared upgrade file sample.py

**Review by gabdank — APPROVED** [review]

**Pull request merged**